src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -26,6 +26,12 @@
     protected $unavailable_commands = [];
 
     protected $valid_frameworks = [];
+    protected $framework_labels = [
```

Comment:
As much as I like having these names printed nicely, it seems like we're creeping into the child classes by putting info on them here.